VirtoCommerce.Storefront.Tests/Model/MoneyOperationTests.cs

```diff
@@ -37,5 +38,26 @@ public class MoneyOperationTests
             Assert.Equal(expectedFormattedAmountWithoutPoint, money.FormattedAmountWithoutPoint);
             Assert.Equal(expectedFormattedAmountWithoutPointAndCurrency, money.FormattedAmountWithoutPointAndCurrency);
         }
+
+        [Theory]
+        // Jira VP-2194 
+        [InlineData(4.6, 4.6, 1)]
```

Comment:
I think here we should check specific array values.
4.6, 1 => [4.6]
10, 3 => [3.34, 3.33, 3.33]
All these values should be checked because otherwise it could be [0,0,10] - it is not what we want.

Reply:
Changed in accordance with comments